plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.h

```diff
@@ -53,6 +53,19 @@ class ColormapOverlay : public kaleidoscope::Plugin {
     overlay_count_ = _overlay_count;
   }
 
+  template<uint8_t _layer_count>
+  void configureOverlays(uint8_t const (&overlays)[_layer_count][kaleidoscope_internal::device.matrix_rows * kaleidoscope_internal::device.matrix_columns]) {
+    overlays_      = Overlay[];
+    overlay_count_ = 0;
+    for (int layer_ = 0; layer_ < _layer_count; layer_++) {
+      for (int key_index_ = 0; key_index_ < kaleidoscope_internal::device.matrix_rows * kaleidoscope_internal::device.matrix_columns; key_index_++) {
+        color_index_ = overlays[layer_][key_index_];
+        // TODO(EvyBongers): validate the color index?
```

Comment:
For implementing a way to specify a key to not have the led color overriden, I'm thinking of using `-1` as a value. I'm wondering if there's a need to validate the upper boundary of the palette. The plugin as it currently exist doesn't do that and I believe that Colormap doesn't to this either, so I feel like I should only verify that lower boundary to skip overriding the color.

Reply:
-1 does not feel unreasonable to me, as long as we define it as a const rather than using a magic number in the code ;)